natlib/svcd.c

```diff
@@ -15,6 +16,8 @@
 // advert_received, which you may want to hook into
 #define MIN_OPT_LEVEL 2
 #include "lrodefs.h"
+#include <stdint.h>
+#include <inttypes.h>
```

Comment:
You shouldn't have needed to include these, I think they should be already included for you.